src/features/profile/userSlice.js

```diff
@@ -42,7 +82,70 @@ const userSlice = createSlice({
         isClosable: true,
         duration: 3000
     })
-    }
+    },
+    [editUserInfo.pending]: (state) => {
+      state.isLoading = true;
+    },
+    [editUserInfo.fulfilled]: (state, { payload }) => {
+      state.isLoading = false;
+      const updatedUser = payload;
+      const userIndex = state.allUsers.findIndex(({_id})=>_id===updatedUser._id);
+      state.allUsers[userIndex] = payload;
+      toast({
+        title: `Profile updated`,
+        status: "success",
+        position: "top-right",
+        isClosable: true,
+        duration: 3000
+      })
+    },
+    [editUserInfo.rejected]: (state, { payload }) => {
+      state.isLoading = false;
+      state.error = payload;
+      toast({
+        title: `${state.error}`,
+        status: "error",
+        position: "top-right",
+        isClosable: true,
+        duration: 3000
```

Comment:
Here you can add common props directly to the toast container.

Reply:
Will do it